packages/geo/src/lib/map/baselayers-switcher/mini-basemap.component.ts

```diff
@@ -112,31 +112,29 @@ export class MiniBaseMapComponent implements AfterViewInit, OnDestroy {
     this.handleLinkedBaseLayer(layer);
   }
 
-  private handleLinkedBaseLayer(baselayer: Layer) {
-    const linkedLayers = baselayer.options.linkedLayers;
+  private handleLinkedBaseLayer(baselayer: Layer): void {
+    const linkedLayers: LayersLink = baselayer.options.linkedLayers;
     if (!linkedLayers) {
       return;
     }
-    const currentLinkedId = linkedLayers.linkId;
-    const currentLinks = linkedLayers.links;
-    const isParentLayer = currentLinks ? true : false;
-    if (
-      isParentLayer &&
-      currentLinkedId === baselayer.options.linkedLayers.linkId
-    ) {
+    const links: LayersLinkProperties[] = linkedLayers.links;
+    const isParentLayer: boolean = links ? true : false;
+    if (isParentLayer) {
       // search for child layers
-      currentLinks.map((link) => {
-        link.linkedIds.map((linkedId) => {
-          const layerToApply = this.map.layers.find(
-            (l) => l.options.linkedLayers?.linkId === linkedId
+      links.map((link: LayersLinkProperties) => {
+        link.linkedIds.map((linkedId: string) => {
+          const layerToApply: Layer = this.map.layers.find(
+            (layer: Layer) => layer.options.linkedLayers?.linkId === linkedId
           );
           if (layerToApply) {
-            const linkedLayerOptions: any = Object.assign(
+            const linkedLayerOptions: LayerOptions = Object.assign(
               Object.create(layerToApply.options),
               layerToApply.options,
               {
                 zIndex: 9000,
-                visible: true,
+                visible: layerToApply.options.linkedLayers?.showInMiniBaseMap !== undefined
+                ? layerToApply.options.linkedLayers.showInMiniBaseMap
+                : true,
```

Comment:
- Pour améliorer la lisibilité, on pourrait utiliser l'opérateur ["Nullish coalescing"](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) qui permet de renvoyer l'opérande de droit lorsque null ou undefined

```suggestion
                visible: layerToApply.options.linkedLayers?.showInMiniBaseMap ?? true,
```

- Problème d'indentation. 
Est-ce que tu as les extensions de Prettier et Eslint dans VsCode? Si oui, ça devrait formatter automatiquement en sauvegardant.